scripts/e2e-servers.js

```diff
@@ -44,21 +47,70 @@ const startServer = serviceName => new Promise((resolve, reject) => {
     server.stdout.on('data', writeToLogStream);
     server.stderr.on('data', writeToLogStream);
     server.on('close', code => writeToLogStream(`${serviceName} process exited with code ${code}`));
-    resolve();
+    resolve(server);
   } catch (err) {
     reject(err);
   }
 });
 
-const startApi = () => startServer('api');
-const startSentinel = () => startServer('sentinel');
-const startAll = () => Promise.all([startApi(), startSentinel()]);
-
 console.log(`To see service log files:
 
 	tail -f logs/api.e2e.log
 	tail -f logs/sentinel.e2e.log
 
 Starting e2e test services…`);
 
-startAll().then(() => console.log('[e2e] All services started.'));
+const app = express();
```

Comment:
cherry picked this from https://github.com/medic/medic/pull/5822 to enable services restart